src/harden.yml

```
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  name: Hardening tasks
+  become: yes
+  become_method: ansible.builtin.sudo
+# Install Banner after hardening instead of during base.yml setup.
+# The hardening script sets the banner acccording to CIS guidlines.
```

Fix typos and clarify language:
```suggestion
  # Install login banner after hardening instead of during base.yml setup
  # since the hardening script modifies the banner according to CIS guidelines
  # and we want to customize it after that.
```